requirements.txt

```diff
@@ -1,5 +1,4 @@
-python-dotenv
-PyMuPDF
-requests
-fastapi[all]
-python-multipart
\ No newline at end of file
+fastapi==0.104.1
```

Comment:
don't we need `fastapi[all]==0.104.1` ?

Reply:
Looking at this [resource](https://fastapi.tiangolo.com/deployment/versions/) I think this is good enough.